src/Controller/Api/MailArchiveController.php

```diff
@@ -30,13 +31,14 @@ class MailArchiveController extends AbstractController
     public function __construct(
         private readonly EntityRepository $froshMailArchiveRepository,
         private readonly EntityRepository $froshMailArchiveAttachmentRepository,
-        private readonly MailSender $mailSender,
+        #[Autowire(service: MailSender::class)]
```

Comment:
This should be `Frosh\MailArchive\Services\MailSender` explicit, not `Shopware\Core\Content\Mail\Service\MailSender` while there could be more decorations.

Reply:
I have changed the import to use the frosh mailsender